README.md

```diff
@@ -185,25 +170,21 @@ channels:
           $ref: "london.hammersmith.transport._public.tube.passenger.avsc"
 ```
 
-This spec will create 3 topics by prepending the id of the app to the 'owned' topics (channels that use a relative path and start with `_private`, `_protected` or `_public`):
+In SpecMesh terms - this file, and what it represents, is considered to be:
+- a data product (Data Mesh terminology)
+- streaming api
+- a policy, or a contract of shared, private and protected related data that is self governed and available to consumers
+- gitops state capture (as part of a git workflow)
+- part of an ecosystem of related dataflow centric apps that are domain centric (each app captures part of the businesses' functionality)
+
+The spec defines 4 topics. Topics are defined under the `channels` section. 
+The spec defines 3 Topics that are _owned_ by this app. 
+Topics that start with `_private`, `_protected` or `_public` are _owned_ topics. The actual topic names will be prefixed with / namespaced under the app's `id`:
```

Comment:
I'm a bit confused here.  The app's `id` is actually `urn:acme.simple_range:life_enhancer` - note the use of a colon between the last two parts.  Yet the topic name uses periods in the same place. If this is some magic conversion, rather than a mistake, then it needs explaining here.